### PR TITLE
Update media queries to eliminate in-between weirdness

### LINKED ISSUE #42: Keep content within colored area on browser widths between 768px and 960px
Right now, the text, logo, and submit button are spilling out of the frame when the window is < 960px wide and > 768px. 

screen shot 2016-07-08 at 5 32 17 pm

This is also causing a weird jump to happen when the content is scrolled.

crappyhourbug

### PR DESCRIPTION
**Pull request by ilikescience**
Closes #42 

@singram26 please review!

### CODE REVIEW

File: assets/styles/layouts/_global.scss [hunk 1]
```diff
@@ -292,7 +292,7 @@ html {
 
 
 /* Small Devices, Tablets */
-@media only screen and (max-width: 768px) {
+@media only screen and (max-width: 1024px) {
```
Comment (samblackk) [inline assets/styles/layouts/_global.scss]: Can we eliminate using raw media queries`media only screen and (max-width: 1024px)` and use `@include media()` instead?

Comment (ilikescience) [inline assets/styles/layouts/_global.scss]: We can. And we SHOULD! I was just being lazy. Update inbound.

**Comment by ilikescience** [comment]
@singram26 updated.

**Comment by samblackk** [comment]
@ilikescience looks good. thanks! i'll merge when the check passes!

**Pull request merged**